src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/models/rgw-storage-class.model.ts

```diff
@@ -0,0 +1,18 @@
+export interface StorageClass {
+  storage_class: string;
+  endpoint: string;
+  region: string;
+  placement_target: string;
+}
+
+export interface StorageClassDetails {
+  target_path: string;
+  zonegroup_name: string;
+  access_key: string;
+  secret_key: string;
+  multipart_min_part_size: number;
+  multipart_sync_threshold: number;
+  host_style: string;
+}
+
+export const TierType = 'cloud-s3';
```

Comment:
So this is a type and not constant.
```suggestion
export const CLOUD_TIER = 'cloud-s3';
```